library/src/lapack/roclapack_getf2_optimized.cpp

```diff
@@ -40,7 +39,7 @@ ROCSOLVER_KERNEL void __launch_bounds__(GETF2_MAX_THDS)
     // batch instance
     T* A = load_ptr_batch<T>(AA, id, shiftA, strideA);
     rocblas_int* ipiv;
-    if(pivot)
+    if(ipivA != nullptr)
```

Comment:
what is the difference between using the argument `pivot` or transforming nullptr into a boolean?
Are you thinking the code could be somehow better if we use one less parameter?